src/EditorFeatures/CSharpTest/MakeClassAbstract/MakeClassAbstractTests.cs

```diff
@@ -0,0 +1,299 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.MakeClassAbstract;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.MakeClassAbstract
+{
+    public class MakeClassAbstractTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
@Evangelink Can we instead just convert these tests to use the new test framework? Will that be too much work? That will avoid you having to add a public ctor for this code fixer and then suppressing `RS0033`.

Reply:
If https://github.com/dotnet/roslyn/pull/40448/files#r450282581 addresses the CI, it is fine to skip changing the tests for this PR.

Reply:
I have tried (if I got it right that the syntax is the same as for roslyn-analyzers) but I end-up with nuget errors: `NuGet.Protocol.Core.Types.FatalProtocolException : Unable to load the service index for source https://api.nuget.org/v3/index.json.`

Reply:
I think it is fine to not do this as part of this PR. You may want to file a follow-up issue with your questions. Also suggest looking at the existing tests which are already moved to new framework, for example: https://github.com/dotnet/roslyn/blob/master/src/Analyzers/CSharp/Tests/RemoveUnnecessaryImports/RemoveUnnecessaryImportsTests.cs